app/models/solution.rb

```diff
@@ -120,19 +120,24 @@ def update_mentoring_status!
     update(mentoring_status: new_status) if mentoring_status != new_status
   end
 
-  # TODO: Erik
+  # TODO
+  def num_views
+    1270
+  end
+
+  # TODO
   def num_loc
-    9
+    18
```

Comment:
Just a side note: how do we want to implement this? Using [cloc](https://github.com/AlDanial/cloc) or should tracks implement this themselves? Or do we just have line count?

Reply:
Yeah, `cloc` running via Lambda could work!

Reply:
https://github.com/exercism/website/issues/668